### PR TITLE
lowercase

### PR DESCRIPTION
**Pull request by Favy-source**
filenames has been changed to lowercases @konfirmed

### CODE REVIEW

**Review by konfirmed — COMMENTED** [review]
Just so we can track stuff better, please let each PR be specific to a task. I see more than lowercase tasks in here @Favy-source

File: theme.config.tsx [hunk 1]
```diff
@@ -16,31 +16,49 @@ const config: DocsThemeConfig = {
...
+      <meta property="og:type" content="website" />
+      <meta property="og:locale" content="en_CA" />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:site" content="@igboamaka_ca" />
+      <meta name="twitter:creator" content="@igboamaka_ca" />
+      <meta name="twitter:title" content={frontMatter.title || 'Igbo'} />
+      <meta name="twitter:description" content={frontMatter.description || 'Your online destination for information about the rich culture and traditions of the Igbo people.'} />
+      <meta name="twitter:image" content="https://igboamaka.ca/logo.png" />
+      <meta name="twitter:image:alt" content="Igboamaka.ca Logo" />
+      <link rel="icon" href="/favicon.ico" />
```
Comment (konfirmed) [inline theme.config.tsx:61]: add favicon

File: theme.config.tsx [hunk 2]
```diff
@@ -16,31 +16,49 @@ const config: DocsThemeConfig = {
...
+    text: 'Igbokwenu.ca is a work in progress. Please check back often for updates.',
+  },
   head: () => {
     const { asPath } = useRouter()
     const { frontMatter } = useConfig()
     return <>
       <meta property="og:url" content={`https://igboamaka.ca${asPath}`} />
       <meta property="og:title" content={frontMatter.title || 'Igbo'} />
       <meta property="og:description" content={frontMatter.description || 'Your online destination for information about the rich culture and traditions of the Igbo people.'} />
+      <meta property="og:image" content="https://igboamaka.ca/logo.png" />
```
Comment (konfirmed) [inline theme.config.tsx:48]: you can use the site logo as the ico. resize it manually first to 32x32 or 40x40. then rename it favicon.ico

File: theme.config.tsx (hunk 1 above)
Comment (Favy-source) [reply theme.config.tsx:61]: Alright sir @konfirmed

**Pull request closed without merging**